### ISSUE TITLE
Compatibility CUDA 5.5

### ISSUE THREAD

**Issue by Anesh20** [flatironinstitute/cufinufft#109]
1) Will cufinufft compile under CUDA 5.5 and gcc 4.4?
2) Also do I need python in order to compile the package? If seems like this based on the readme file.
Thanks!

**Comment by MelodyShih** [comment]
Hi Anesh20, 

1. Yes, it will compile under CUDA 5.5 + GCC 4.4.
2. No. You don't need it if you are not using the python interface. 

Are you facing any problem installing the library?

**Comment by Anesh20** [comment]
Hi Melody,
Yes, I tried to compile and got this error msg:
"nvcc fatal : unknown option 'std' "
Thanks
Anesh

**Comment by garrettwrong** [comment]
5.5 is quite old (currently cuda is 11ish..). I don't recall building
against anything before 8, and I think that required a minor tweak...  Iirc
major release 5 will be limited to Kepler and prior cards ( so six and
prior years ago or so)...

I think it is safe to remove flags related to c and c++ standards from the
Makefile in this case. Likely it is complaining about that. Unfortunately I
don't have anything that old to test against. You may find the performance
is not inspiring on older cards.  Please let us know what you find. Thanks

**Comment by Anesh20** [comment]
I removed all std instances in the Makefile and had to removed other syntaxes not available under 5.5 e.g. -Wno-deprecated-gpu-targets, --default-stream per-thread and sm_50 and greater. 
Now I am stuck with lots of errors ([Output.txt](https://github.com/flatironinstitute/cufinufft/files/6456010/Output.txt)) and am not sure how to proceed.
The Makefile I used is also attached: [Makefile.txt](https://github.com/flatironinstitute/cufinufft/files/6456021/Makefile.txt)
Any advice will be appreciated. Thanks!
